src/controllers/admin.ts

```diff
@@ -70,4 +73,54 @@ const getTherapistsForAdmin = async (req: Request, res: Response, next: NextFunc
   }
 };
 
-export { adminLogin, getTherapistsForAdmin };
+const updateTherapistActive = async (req: Request, res: Response, next: NextFunction) => {
+  try {
+    const { id, active } = req.params;
+    const therapistId = Number(id);
+
+    // eslint-disable-next-line no-restricted-globals
+    if (isNaN(therapistId)) {
+      throw templateErrors.BAD_REQUEST('Therapist id should be a valid number');
+    }
+
+    if (active !== 'true' && active !== 'false') {
+      throw templateErrors.BAD_REQUEST('Active should be a valid boolean');
+    }
+
+    const therapist = await patchTherapist(therapistId, active === 'true');

```

Comment:
what if active is undefined?
So the condition to check active undefined is meaningless somehow

Reply:
I didn't get  it, if active undefined it should throw a BAD_REQUEST